profiler/src/ProfilerEngine/Datadog.Profiler.Native/Configuration.cpp

```diff
@@ -73,6 +73,20 @@ Configuration::Configuration()
     _gitCommitSha = GetEnvironmentValue(EnvironmentVariables::GitCommitSha, DefaultEmptyString);
     _isInternalMetricsEnabled = GetEnvironmentValue(EnvironmentVariables::InternalMetricsEnabled, false);
     _isSystemCallsShieldEnabled = GetEnvironmentValue(EnvironmentVariables::SystemCallsShieldEnabled, true);
+
+    // Check CI Visibility mode
+    _isCIVisibilityEnabled = GetEnvironmentValue(EnvironmentVariables::CIVisibilityEnabled, false);
+    _internalCIVisibilitySpanId = uint64_t{0};
+    if (_isCIVisibilityEnabled)
+    {
+        // We cannot write 0ull instead of std::uint64_t{0} because on Windows, compiling in x64, std::uint64_t == unsigned long long.
+        // But on Linux, it's std::uint64_t == unsigned long (=> 0ul)and it fails to compile.
+        // Here we create a 0 value of type std::uint64_t which will succeed the compilation
+        _internalCIVisibilitySpanId = GetEnvironmentValue(EnvironmentVariables::InternalCIVisibilitySpanId, uint64_t{0});
```

Comment:
nit (can be done in another review)
move it outside the `if` and replace line 79 by it.